pkg/packagekit/updates.jsx

```diff
@@ -56,18 +56,43 @@ const PK_STATUS_STRINGS = {
     14: _("Verifying"),
 }
 
+const I_TRANS = "org.freedesktop.PackageKit.Transaction";
```

Comment:
caps> old habit for naming constants from C programming. The various `PK_{EXIT,INFO}_*` constants are also all caps as they are defined like that in the [C headers](https://github.com/hughsie/PackageKit/blob/master/lib/packagekit-glib2/pk-enum.h).

I use it about 10 times, and wanted to keep it shourt. But happy to rename to `transactionInterface` - it seems there is no real convention in JS for naming constants. Fixed locally (will push once I addressed everything else).